build/Jenkinsfile.pr

```diff
@@ -125,14 +125,22 @@ pipeline {
                 withDockerContainer(args: "-u root --privileged --net ${COMPOSE_PROJECT_NAME}_mm-test -v ${WORKSPACE}/src:/go/src/", image: 'mattermost/mattermost-build-server:dec-7-2018') {
                     ansiColor('xterm') {
                     sh """
-                            cd /go/src/github.com/mattermost/mattermost-server
+                        cd /go/src/github.com/mattermost/mattermost-server
+
+                        mkdir -p client/plugins
+                        cat config/config.json
 
-                            mkdir -p client/plugins
-                            cat config/config.json
+                        make test-server BUILD_NUMBER='${BRANCH_NAME}-${BUILD_NUMBER}' TESTFLAGS= TESTFLAGSEE=
 
-                            make test-server BUILD_NUMBER='${BRANCH_NAME}-${BUILD_NUMBER}' TESTFLAGS= TESTFLAGSEE=
                         """
                     }
+                    withCredentials([string(credentialsId: 'CODECOV_TOKEN', variable: 'CODECOV')]) {
+                        sh '''#!/bin/bash
```

Comment:
Why the shebang here but not above? Anyway, as per https://jenkins.io/doc/pipeline/steps/workflow-durable-task-step/#sh-shell-script, bash is the default, as is `set -xe`.

Reply:
I would like to keep this extra `sh` because it will be an extra step in the jenkins UI and then will be more easy to see the logs if the codecov failed to upload

Reply:
No objection to the `sh` for the extra step, but my comment was specifically aobut the shebang `#!/bin/bash` and the `set -x`, both of which are redundant as per the above link.